packages/bierzo-wallet/src/routes/account/index.ts

```diff
@@ -3,7 +3,7 @@ import AccountManage from "./manage";
 import AccountRegister from "./register";
 import AccountUpdate from "./update";
 
-export type AccountEntity = "iovname" | "starname" | "name";
+export type AccountEntity = "iovname" | "starname" | "name" | "name-back";
```

Comment:
I think is better to create new type extending this type and put it into `packages/bierzo-wallet/src/routes/account/transfer/index.tsx`. Because this additional string literal used only in this component.